turnitintooltwo_submission.class.php

```diff
@@ -180,7 +180,7 @@ private function get_submission_details($idtype = "moodle", $turnitintooltwoassi
                 $this->turnitintooltwoid = $turnitintooltwoassignment->turnitintooltwo->id;
             }
 
-            if (count($turnitintooltwoassignment->get_parts() > 1)) {
+            if (count($turnitintooltwoassignment->get_parts()) > 1) {
```

Comment:
I agree that this is an obvious typo, but I'm concerned that this actually break the functionality without additional changes.

Specifically: because this code always returned true before, it does not seem to correctly handle the case where the number of "parts" is equal to 1. In situations where there is only one "part", it seems like this warning-free code will not initialize the overall grade member variable.

If the code does not need to be run when there is only one part, then it is correct, but there is not a comment to clarify what the right behavior is. Otherwise, I suggest:
```php
$parts = $turnitintooltwoassignment->get_parts();
if (!empty($parts)) {
```

Hope that helps!

Reply:
are you guys going to fix it yourself or you want me to provide an additional patch as the first one was merged?

Reply:
Hi @dmitriim, we are going to take a look at this. If it requires more work we will make a patch as we're close to releasing.

Reply:
After looking into this, I think this is fine as it is - if there is only one part then we don't have an overall grade.